### PR TITLE
Documentation Updates (Collect Logs Action)

### PR DESCRIPTION
**Pull request by kkeirstead**

### CODE REVIEW

File: documentation/configuration.md [hunk 1]
```diff
@@ -674,6 +674,32 @@ Usage that collects a CPU trace for 1 minute and egresses it to a provider named
...
+An action that collects logs for the process that the collection rule is targeting.
+
+##### Properties
+
+| Name | Type | Required | Description | Default Value | Min Value | Max Value |
+|---|---|---|---|---|---|---|
+| `DefaultLevel` | [LogLevel](api/definitions.md#LogLevel)? | false | The default log level at which logs are collected for entries in the FilterSpecs that do not have a specified LogLevel value. | `LogLevel.Warning` | | |
+| `FilterSpecs` | Dictionary<string, [LogLevel](api/definitions.md#LogLevel)?> | false | A custom mapping of logger categories to log levels that describes at what level a log statement that matches one of the given categories should be captured. | `null` | | |
+| `UseAppFilters` | bool | false | Specifies whether to capture log statements at the levels as specified in the application-defined filters. | `true` | | |
+| `Format` | [LogFormat](api/definitions.md#LogFormat)? | false | The format of the logs artifact. | `LogFormat.EventStream` | | |
```
Comment (jander-msft) [inline documentation/configuration.md:688]:
```suggestion
| `Format` | [LogFormat](api/definitions.md#LogFormat)? | false | The format of the logs artifact. | `EventStream` | | |
```

File: documentation/configuration.md [hunk 2]
```diff
@@ -674,6 +674,32 @@ Usage that collects a CPU trace for 1 minute and egresses it to a provider named
...
+| `Duration` | TimeSpan? | false | The duration of the logs operation. | `"00:00:30"` (30 seconds) | `"00:00:01"` (1 second) | `"1.00:00:00"` (1 day) |
+| `Egress` | string | true | The named [egress provider](egress.md) for egressing the collected logs. | | | |
+
+##### Example
+
+Usage that collects logs at the Information level for 30 seconds and egresses it to a provider named "TmpDir".
```
Comment (jander-msft) [inline documentation/configuration.md:694]: That's not correct. This example collects logs as specified in the application-defined filters and falls back to Interformation level for any defined category that does not have a specified level.

Comment (jander-msft) [reply documentation/configuration.md:694]: If you want to keep the explanation as-is, you'd want to update the example to something like:

```json
{
  "DefaultLevel": "Information",
  "UseAppFilters": false,
  "Egress": "TmpDir"
}
```

File: documentation/configuration.md [hunk 3]
```diff
@@ -718,4 +744,4 @@ The following example shows the `Limits` portion of a collection rule that has t
         "ActionCountSlidingWindowDuration": "01:00:00",
     }
 }
-```
\ No newline at end of file
+```
```
Comment (jander-msft) [inline documentation/configuration.md:747]: What changed here?

Comment (kkeirstead) [reply documentation/configuration.md:747]: I don't _think_ I changed anything there; I figured that was just from adding new lines in, so the text around it shifted (it's also highlighted in definitions.md).

**Pull request merged**